test/e2e/performanceprofile/functests/utils/namespaces/namespaces.go

```diff
@@ -30,6 +30,9 @@ func init() {
 var TestingNamespace = &corev1.Namespace{
 	ObjectMeta: metav1.ObjectMeta{
 		Name: testutils.NamespaceTesting,
+		Annotations: map[string]string{
```

Comment:
since this is mixed-cpus feature related only and is not needed by default. do we verify it somewhere in the mixedcpus suite? I understand the functional tests that require workload will fail, but I am trying t find out if this specific detail is covered in testing

Reply:
We can add negative test - deploy a pod in a namespace without the annotation and expect admission error, is that ok?

Reply:
will the feature still need this annotation when in GA?
unless this annotation will affect non mixed-cpus tests, I think no harm to keep it here by default. otherwise, I'd just preserve the default ns and adjust it in the suite when needed.
adding the negative test sounds good and is part of the TP which can be added later, no rush.

Reply:
AFAIK the annotation would stay.
And it won't affect non mixed CPUs workload